purrsoft/.env

```diff
@@ -1 +1 @@
-VITE_API_PATH=https://localhost:7233/api
\ No newline at end of file
+VITE_API_PATH=http://localhost:5021/api
```

Comment:
Why change the api point?
Last time I checked on backend the 7233 is set for https.

Reply:
And why go back to http?

Reply:
It did not work on 7233 or https. When I started the server, it started on ip http://localhost:5021 and I have the latest commit of main.

Reply:
today i started the server and the ip changed back to https and 7233, i do not know what happened yesterday :))